pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -271,9 +270,7 @@ stdenv.mkDerivation rec {
     gperf
     ninja
     meson
-    coreutils # meson calls date, stat etc.
     glibcLocales
-    patchelf
```

Comment:
Mind explaining why we don't need it anymore? Was its usage removed (when?) but this line left?

Reply:
It is included in stdenv. This line does nothing.